frontend/app/routes/_gcweb-app.view-letters._index.tsx

```diff
@@ -58,6 +59,7 @@ interface LetterListProps {
 }
 
 function LetterList({ letters }: LetterListProps) {
+  const { filePath } = useLoaderData<LoaderData>();
```

Comment:
```suggestion
  const { filePath } = useLoaderData<<typeof loader>>();
```

This suggestion will remove the need to define the `LoaderData` type above.